specification/dimensions/commitmentname.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Name
+
+A commitment-based discount is a contractual commitment to use a certain amount of a specific type of usage for a fixed term in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
+
+A Commitment Name is the display name assigned to a commitment-based discount.
+
+The CommitmentName column MUST be present in the billing data. This column must be of type String. The CommitmentName value MAY be null if a display name cannot be assigned to a commitment-based discount. CommitmentName MUST NOT be null if a display name can be assigned to a commitment-based discount.
```

Comment:
Does it add any value to say that it does not need to be unique? ie. what do FinOps Practioners use this field for?